src/dotnet-scaffolding/dotnet-scaffold/Flow/Steps/CategoryPickerFlowStep.cs

```diff
@@ -92,15 +93,24 @@ public ValueTask<FlowStepResult> ValidateUserInputAsync(IFlowContext context, Ca
 
             SelectComponent(context, dotnetToolComponent);
             SelectCommand(context, commandInfo);
-            SelectCategory(context, commandInfo.DisplayCategory);
+            SelectCategories(context, commandInfo.DisplayCategories);
             return new ValueTask<FlowStepResult>(FlowStepResult.Success);
         }
 
-        private void SelectCategory(IFlowContext context, string categoryName)
+        private void SelectCategories(IFlowContext context, List<string> categories)
         {
             context.Set(new FlowProperty(
-                FlowContextProperties.ScaffoldingCategory,
-                categoryName,
+                FlowContextProperties.ScaffoldingCategories,
+                categories,
+                "Scaffolding Categories",
```

Comment:
Where does this line show up in the UI? I'm wondering if we need 'Scaffolding' here. The user is already in `dotnet scaffold` so it may be redundant.

Reply:
does not show up in the UI?

Reply:
currently there is no display of the selected options.

Reply:
ok thanks